libs/utils/wa_results_collector.py

```diff
@@ -458,11 +463,17 @@ def _get_extra_job_metrics(self, job_dir, workload):
         # think we can simplify this.
         for artifact_name, path in artifacts.iteritems():
             if os.stat(path).st_size == 0:
-                self._log.info(" no data for %s",  path)
+                #self._log.info(" no data for %s",  path)
                 continue
 
             if artifact_name.startswith('energy_instrument_output'):
-                df = pd.read_csv(path)
+                #self._log.info(" power file %s",  path)
```

Comment:
Stray comment ?

Reply:
yes. a debug trace that should have been removed